image/scripts/install_dependencies.sh

```diff
@@ -72,6 +67,23 @@ if [ "$PKGSTOINSTALL" != "" ]; then
     apt-get -y clean
 fi
 
+
+# Things to get rid of. Factor out to file if many turn up.
+grep -w "ii  deja-dup" /tmp/installed-package-list.txt > /dev/null
```

Comment:
This script should actually be tested with an os2borgerpc installation while it is now used for both types of pc installations.

Reply:
True!